rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -83,6 +83,11 @@ static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version,
 	vm::ptr<CellSaveDataFileSet>  fileSet  = g_savedata_context.ptr(&savedata_context::fileSet);
 
 	// path of the specified user (00000001 by default)
+	if (userId == CELL_SYSUTIL_USERID_CURRENT)
```

Comment:
Would you prefer to remove the step for retrieving the current ID, where the TODO is. Or remove the check for 0 in the string initialisation?